server/src/logic/services/user.service.ts

```diff
@@ -0,0 +1,27 @@
+import { IDatabase } from "pg-promise";
+import { SocoboUser } from "./../../models/socobouser";
+
+
+export class UserService {
+  private _db: IDatabase<any>;
+
+  constructor (db: IDatabase<any>) {
+    this._db = db;
+  }
+
+  getAllUsers (): Promise<SocoboUser[]> {
+    return new Promise((resolve, reject) => {
```

Comment:
I think you don't need to create a new Promise because the many function already returns a promise